src/components/Account/OAuthLogin/OAuthLogin.container.js

```diff
@@ -54,11 +60,20 @@ class OAuthLoginContainer extends React.Component {
     return !!user.email;
   }
 
+  handleError() {
+    const { history } = this.props;
+    this.setState({ hasErrors: true });
+    setTimeout(() => {
```

Comment:
for me. is not necessary to stop the user interface to show the error message. probably we could redirect to /login-signup directly and show the error message on a notification or modal.